src/GenericSpectator/Assets/Scenes/SpectatorView.GenericObserver.unity

```diff
@@ -0,0 +1,252 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Hmm... I should probably change the name of this scene to match the project name better.  Perhaps "SpectatorView.GenericSpectator" assuming the project name stays of that form.

Reply:
if its a standalone project, i've seen "Main" used as well

Reply:
That's a good idea.  I'll probably switch to that.

---
In reply to: [337259015](https://github.com/microsoft/MixedReality-SpectatorView/pull/250#discussion_r337259015) [](ancestors = 337259015)